.github/workflows/release.yml

```diff
@@ -31,7 +31,10 @@ jobs:
       - name: Package
         run: |
           mvn clean package -DskipTests
-          mv "$(find hugegraph-*.tar.gz | head -1)" hugegraph-${TAG_NAME/#v/}.tar.gz
+
+      - name: Rename file
```

Comment:
why need to split into 2 stages? can put "mvn clean package -DskipTests" into "Rename file" stage?

Reply:
because of the "Rename file" stage is not unessential

Reply:
OK